src/scss/components/header/_cart.scss

```diff
@@ -0,0 +1,205 @@
+
+.header__cart {
+  float: right;
+
+  .cart-info, &-btn, .shopping-cart {
+    display: none;
+  }
+
+  &-icon {
+    display: inline-block;
+    height: 30px;
+    position: relative;
+
+    svg {
+      fill: $gray;
+      height: 30px;
+      width: 30px;
+      vertical-align: middle;
+    }
+
+    .cart-products-number {
+      color: $tertiary;
+      font-family: $franklin-medium-cd;
+      font-size: 12px;
+      left: 9px;
+      line-height: 1;
+      position: absolute;
+      text-align: center;
+      top: 5px;
+      width: 14px;
+    }
+  }
+}
+
+@media (min-width: 48em) {
```

Comment:
I'm proposing to change those variables to em to not calculate like $mobile + 1 to have 768px